src/lib/docs/get-parent-root-docs-path.ts

```diff
@@ -0,0 +1,30 @@
+import { RootDocsPath } from 'types/products'
+
+export function getParentRootDocsPath(
```

Comment:
Split out this utility. Already existing in almost exactly this form in `get-back-to-link`, and I figured since I needed the exact same functionality in `get-docs-breadcrumbs` it was worth splitting out & writing tests.